scripts/sct_compute_mtr.py

```diff
@@ -45,22 +45,32 @@ def get_parser():
         )
 
     optional = parser.add_argument_group("\nOPTIONAL ARGUMENTS")
+    optional.add_argument(
+        "-thr",
+        type=float,
+        help="Threshold to clip MTR output values in case of division by small number. This threshold is applied on the"
+             "absolute value. Default: 100.",
+        default=100
```

Comment:
So values will be between -100 and 100, is it right? or am I misunderstanding?
If so, could it be helpful to mention this in the help?

Reply:
agreed-- clarified in 509c99b2ee3d4659d61156a34912f6fd4497f993

Reply:
great!